timelock-impl/src/main/java/com/palantir/atlasdb/backup/AtlasBackupResource.java

```diff
@@ -111,6 +111,28 @@ private InProgressBackupToken prepareBackup(Namespace namespace) {
                 MoreExecutors.directExecutor());
     }
 
+    @Override
+    public ListenableFuture<Void> disableTimelock(AuthHeader authHeader, Set<Namespace> namespaces) {
+        return handleExceptions(() -> disableInternal(namespaces));
+    }
+
+    private ListenableFuture<Void> disableInternal(Set<Namespace> namespaces) {
+        // todo(gmaretic): disable all remote nodes
+        // todo(gmaretic): disable locally
+        return Futures.immediateVoidFuture();
+    }
+
+    @Override
+    public ListenableFuture<Void> reenableTimelock(AuthHeader authHeader, Set<Namespace> namespaces) {
```

Comment:
thoughts on combining this with `completeRestore`, to minimise the number of calls the internal product needs to make to Atlas?